lib/variables/variable.hpp

```diff
@@ -17,27 +17,28 @@ class Variable : public Item {
 public:
 	Variable();
 	~Variable();
-	void Load(obs_data_t *obj);
-	void Save(obs_data_t *obj) const;
-	EXPORT std::string Value(bool updateLastUsed = true) const;
-	EXPORT std::optional<double> DoubleValue() const;
-	EXPORT std::optional<int> IntValue() const;
-	EXPORT std::string PreviousValue() const;
-	void SetValue(const std::string &value);
-	void SetValue(double value);
-	std::string GetDefaultValue() const { return _defaultValue; }
 	static std::shared_ptr<Item> Create()
 	{
 		return std::make_shared<Variable>();
 	}
 
-	enum class SaveAction {
+    enum class SaveAction {
```

Comment:
Can you please fix the indentation here?
Not sure why this is not caught by `Check Formatting`.